documentation/development-processes-checklists.md

```diff
@@ -91,7 +103,7 @@ Upon completion of the testing phase, conduct the release procedure (outlined be
         - Include summary of `Merge hotfix/359 (#361) from master`.
         - Include the same description in the second line.
 
-### Release Procedures
+### Release Procedure
```

Comment:
Either way works, but we now have `Sprint Closeout Proecedures`, `Development Procedures`, and `Release Procedures`. Shouldn't these have a consistent pluralization?